app/furniture/marketplace/delivery.rb

```diff
@@ -1,12 +1,23 @@
 class Marketplace
   class Delivery < Record
     self.table_name = "marketplace_orders"
+
     belongs_to :marketplace
     belongs_to :shopper
+    belongs_to :delivery_area
+    validates :delivery_area_id, presence: true # rubocop:disable Rails/RedundantPresenceValidationOnBelongsTo
```

Comment:
How come we need to validate this separately? Is it because it is somehow not a "real" association, or some other not-quite-standardness? (Asking for my own enlightenment.)

Reply:
I was annoyed and confused by this to, but because we set the `delivery_area_id` in the form if I don't have it then there's no feedback about why it's invalid.

Reply:
Ugh. Fair enough.